seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-file/src/main/scala/org/apache/seatunnel/spark/source/File.scala

```diff
@@ -30,7 +33,15 @@ import scala.util.{Failure, Success, Try}
 class File extends SparkBatchSource {
 
   override def checkConfig(): CheckResult = {
-    checkAllExists(config, PATH, FORMAT)
+    checkAllExists(config, PATH)
+  }
+
+  override def prepare(prepareEnv: SparkEnvironment): Unit = {
+    val defaultConfig = ConfigFactory.parseMap(
+      Map(
+        FORMAT -> DEFAULT_FORMAT
```

Comment:
Seem like add default format is JSON, why not TEXT? TEXT can support any file data.

Reply:
I agree, but the official documentation show json